src/webui/static/app/dvr.js

```diff
@@ -119,27 +119,41 @@ tvheadend.dvrDetails = function(uuid) {
         var buttons = [];
 
         buttons.push(new Ext.Button({
+            disabled: !title,
             handler: searchIMDB,
             iconCls: 'imdb',
             tooltip: _('Search IMDB (for title)'),
         }));
 
         buttons.push(new Ext.Button({
+            disabled: !title,
             handler: searchTheTVDB,
             iconCls: 'thetvdb',
             tooltip: _('Search TheTVDB (for title)'),
         }));
 
+        buttons.push(new Ext.Button({
+            disabled: !title,
+            handler: searchFilmAffinity,
+            iconCls: 'filmaffinity',
+            tooltip: _('Search FilmAffinity (for title)'),
+        }));
+
         function searchIMDB() {
             window.open('http://akas.imdb.com/find?q=' +
                         encodeURIComponent(title), '_blank');
         }
 
-        function searchTheTVDB(){
+        function searchTheTVDB() {
             window.open('http://thetvdb.com/?string='+
                         encodeURIComponent(title)+'&searchseriesid=&tab=listseries&function=Search','_blank');
         }
 
+        function searchFilmAffinity() {
+            window.open('https://www.filmaffinity.com/es/search.php?stext=' +
```

Comment:
is it a good choice to hard coding the spanish version ?

Reply:
My fault, I will add a language check to use the English or Spanish version.

Reply:
IMO, I would just use en & not bother with any checks. The reasons for this are..

- People will ask  "why isn't <insert lang here> supported in tvheadend, it's supported on the site."
- The available languages (on the site) may change at any point.

Reply:
Filmaffinity it's a spanish page, it's support two languages (english and spanish), by default now it's locale dependent, so will use spanish version only if you set it as language in user config.